src/Hosting/Hosting/src/Internal/HostingApplicationDiagnostics.cs

```diff
@@ -16,9 +16,9 @@ internal class HostingApplicationDiagnostics
     {
         private static readonly double TimestampToTicks = TimeSpan.TicksPerSecond / (double)Stopwatch.Frequency;
 
-        private const string ActivityName = "Microsoft.AspNetCore.Hosting.HttpRequestIn";
-        private const string ActivityStartKey = ActivityName + ".Start";
-        private const string ActivityStopKey = ActivityName + ".Stop";
+        public static readonly string ActivityName = "Microsoft.AspNetCore.Hosting.HttpRequestIn";
+        private static readonly string ActivityStartKey = ActivityName + ".Start";
+        private static readonly string ActivityStopKey = ActivityName + ".Stop";
```

Comment:
Add a comment about why these are static readonly instead of const.

Reply:
I don't know why they changed 😆
@shirhatti

Reply:
I wanted to do object equality in the test, but it's no longer needed. Your current test fix is better